src/auth/ntlm/SMB_LM/ntlm_smb_lm_auth.cc

```diff
@@ -93,7 +93,7 @@ static unsigned char challenge[NTLM_NONCE_LEN];
 static unsigned char lmencoded_empty_pass[ENCODED_PASS_LEN],
        ntencoded_empty_pass[ENCODED_PASS_LEN];
 SMB_Handle_Type handle = NULL;
-int ntlm_errno;
+NtlmError ntlm_errno;
```

Comment:
If possible, to avoid questions about this global being (not) shared with other translation units:

```suggestion
static NtlmError ntlm_errno;
```

I do not insist on this change.

Reply:
It won't hurt so why not?